### DISCUSSION TITLE
Pondering Data, Information, and Knowledge

### DISCUSSION THREAD

**Discussion by orcmid** [amyjko/foundations-of-information#14] [General]
I am always baffled by striking difference between what some things are versus what we have them mean.  There are layers of that, so it becomes a peculiar effort.  Here is what I wrestle with.  I don't know how it might be helpful in the treatment of Data, Information, and Knowledge.

With regard to data and programs (data interpreted as scripts, as it were), I satisfy myself by expressing an intended interpretation, what I claim to have represented in a lower/different level by some arrangement.  We can say the same thing about texts and the intended interpretation that an author may have wanted to convey.  For me, the intended meaning is not inherent in the data by which something is conveyed.  The discussion of "00000001" is exemplary.

My impression of Shannon's extraordinary treatment is that one can look at the degree to which a communication is compressible without loss of the intended interpretation, provided you know the scheme.  What's interesting about that is a truly random sequence of bits has the ability to guess the next bit be undistinguishable from 50-50, no matter that we have seen all the preceding bits.  And with regard to Shannon's interest in cryptographic communication, that's ideal for cipher texts and he identifies a scheme for achieving it.  

An interesting contemporary case is with JPEG encoding of images.  As typically used, this involves a lossy compression.  The exact bits of the image data that was subjected to JPEG encoding are not recoverable.  Yet, there is a degree to which intended interpretation as an image is satisfactory (until it isn't).  Extracting a CSV format from an Excel (or other) spreadsheet is a similar act.  So is extracting the plaintext of a richly-formatted document.

I am using examples from data processing (before it was elevated to information processing) because digital systems seem to be the arena in which we have come to struggle about who is doing what in acts of interpretation and what is or is not inherent to the literal thing.

To the degree to which it is us who interprets data as communications, and what the communication is, it is useful to avoid having something be inherent in data that is not intrinsic yet interpretable in a situated context.

This has me think that knowledge does not enter here very quietly.  I am thinking that knowledge is very tied to belief, and that strikes me as always more involved than in any direct sense of how something is taken as assertions of facts.  Of course, that interpretation as fact might be intended, but contrary interpretations are also found.  I think confirmation bias is an useful warning flasher.  And now there is concern about algorithmic bias and (very weird for me), ethical things (AIs, algorithms, other artifacts).   I think I just got lost in a maze of my own creation.

With regard to emotion and also what is evocative for us, I think the Christopher Alexander treatment of Pattern Languages is interesting and also a bit troubling.  I don't mean the reduction in software development but Alexander's investment of qualities in the components and constructions of architected habitats.  Donald Norman came to use the term affordance as something we recognize in an artifact.  It is some quality that is evocative for us and a producer will make provisions that afford that.  Of course, we are also trained to what is evocative and directly appealing.

I'll stop musing for now.

[PS: Reading the last parts of the information chapter has me thing that, whatever my concerns, I also have violent agreement :]

**Comment by orcmid** [comment]
Here's something that I received from my colleague, William Anderson, the other day.  It seems very relevant here.  I am in the zipper school on some matters, but not in how we create meaning automatically.

  -----

Conrad believed that "the general effect of a novel must be the general effect that life makes on mankind." A person doesn't process things as "a narration, a report," lining up observations and interpretations like two sides of a zipper. Instead you absorb impressions, feelings, and sensations. Maybe you can spot a pattern, maybe not; maybe you only recognize things much later. As it is, what you see on the face of things may or may not tell you much about what it means -- the same way you can be hypnotized by the play of light and shadow on pool of water without ever knowing how cold or salty it may be.

Maya Jasanoff.  p. 147 in "The Dawn Watch: Joseph Conrad in a Global World" ISBN: 978-1-59420-581-1 (http://www.worldcat.org/oclc/1150917684)
(An exploration of the life of Joseph Conrad and his own turbulent age of globalization that reveals much about the heart of darkness of our own time.)

**Comment by amyjko** [comment]
There is indeed a lot to ponder in this chapter! I hope it conveyed the lack of certainty and agreement on the subject. There's still much in play about the boundaries between these three ideas and where they live in reality (or whether they do at all). I'm sure every academic discipline has something to say on the matter, but few agree. What a deep irony that one of the most central features of 21st century life is one so poorly understood.

**Reply by orcmid** [reply]
Yes.  I think this is one of those "cat trying to explain to a fish what it is like to be wet."  I first ran into this in a 1960s statement by Saul Gorn: "Trying to explain programming languages to a programmer is like a ... ".

**Comment by orcmid** [comment]
[Aside: I suspect stenography is more difficult in a JPEG than a loss-less compression, though must still be indistinguishable from low-order noise.  Of course, the ancients found ways to secrete messages in their art.]

I think ontology is relevant here.  I don't mean just taxonomies of some form, some of which are not ontological very much at all.

I mean ontological in the sense of granting being to something, via language in some manner probably.  The taxonomical classifications of life on Earth do lead to ontological commitments.  We take as given and have beliefs about species, classes, and varieties.  We grant being to "race" even though there is apparently no undeniable biological basis for it.  And of course the struggling around sex, sexuality, and gender.  On safer ground, we grant being to the Pacific Ocean, the Puget Sound, Lake Washington etc.  Also Washington State, King County, City of Seattle, and The University of Washington.  Speaking of computers or (running) programs as animate stems from a pretty-human trait.  Some of these, perhaps many, are commonplace, some are troublesome.

One aspect of ontological commitment is that it closes off other ways of looking, makes other distinctions difficult to frame.  I suspect a gentle way to consider ontological distinctions is as providing useful ways to look at something, but not make them so paradigmatic that it all goes unreal.  (Dennis notices that he has made an ontological commitment about ontological commitment.)

How might this be relevant in the context of data, information, and knowledge? I think it is technically closer to information than knowledge (which is therefore vulnerable to it).  I also think it is relevant in discussions about what something is taken to be about.  In formulating what we call information systems, we need to avoid not only confusion of the map with the territory, but also computational structures with entities in nature and society.  (The semantic web comes to mind as an unfortunate example.)

Hmm, have to figure out how pull the soap-box out from under this.

**Reply by amyjko** [reply]
Your points above are related to this:

https://github.com/amyjko/foundations-of-information/issues/15

which is about citing this:

Bowker, G. C., & Star, S. L. (2000). Sorting things out: Classification and its consequences.

That's one of the seminal texts on classification, ontology, and it's social consequences.

**Reply by orcmid** [reply]
Thanks for that.  I don't have access to the book but G.C. Bowker's faculty page has valuable excerpts.

I guess we need to think about metaphysics enough to hope that's not what we're doing.  When someone goes from ontological commitment to assertion of indelible truth, I think that's a metaphysical claim, the kind of identification of belief/faith with truth that seems to arise, whether it is clinging to the infallibility of science or that of scripture.
